### PR TITLE
feat: label support for directories

### PR DESCRIPTION
**Pull request by lechnerc77**
## Purpose

* This PR introduces label support for directories

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code via unit tests:

```
go test ./...
```
* Alternative: Build Terraform provider and create a directory with or without labels 

## What to Check

Verify that the following are valid

* Directory gets created as configured

## Other Information

An additional test for labels cannot be created, dependency to PR #219

### CODE REVIEW

File: internal/btpcli/facade_accounts_directory_test.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func TestAccountsDirectoryFacade_Create(t *testing.T) {
 			assertCall(t, r, command, ActionCreate, map[string]string{
 				"globalAccount": "795b53bb-a3f0-4769-adf0-26173282a975",
 				"displayName":   "my-directory",
+				"labels":        "null",
```
Comment (v0lkc) [inline internal/btpcli/facade_accounts_directory_test.go:52]: I think in this case, labels shouldn't be set at all

Comment (v0lkc) [inline internal/btpcli/facade_accounts_directory_test.go]:
```suggestion
```

I think in this case it's better to not send the labels to the cli backend.

File: internal/btpcli/facade_accounts_directory_test.go [hunk 2]
```diff
@@ -73,6 +74,7 @@ func TestAccountsDirectoryFacade_Create(t *testing.T) {
 				"description":   description,
 				"subdomain":     subdomain,
 				"parentID":      parentId,
+				"labels":        "null",
```
Comment (v0lkc) [inline internal/btpcli/facade_accounts_directory_test.go]: same here

**Review by v0lkc — CHANGES_REQUESTED** [review]

File: internal/btpcli/facade_accounts_directory_test.go (hunk 1 above)
Comment (lechnerc77) [reply internal/btpcli/facade_accounts_directory_test.go]: Fixed

File: internal/btpcli/facade_accounts_directory_test.go (hunk 2 above)
Comment (lechnerc77) [reply internal/btpcli/facade_accounts_directory_test.go]: Fixed

**Review by v0lkc — APPROVED** [review]

**Pull request merged**